code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs

```diff
@@ -125,6 +125,10 @@ si = SI {
 }
   where qDefs = mapMaybe qdEFromDD dataDefs
 
+purp :: Sentence
+purp = foldlSent_ [S "simulate", getAcc twoD, phrase CP.rigidBody, S "for use in",
```

Comment:
Should this use `plural` instead of `phrase`? I don't think the phrase "simulate 2D rigid body" makes sense
```suggestion
purp = foldlSent_ [S "simulate", getAcc twoD, plural CP.rigidBody, S "for use in",
```

Reply:
Ah, thanks for catching this! I meant to write "simulate 2D rigid body *physics*." I will fix this now.

Reply:
Okay, I have committed the fix. Resolving this now.